src/YQWizard.h

```diff
@@ -579,6 +580,7 @@ protected slots:
     static std::string		 _releaseNotesButtonId;
     static std::string		 _releaseNotesButtonLabel;
     YQWizardButton *	_helpButton;
+    QAction *           _helpAction;
```

Comment:
IMHO exposing the QAction here has no value and just perturbs the ABI, doesn't it?